### PR TITLE
yeh

### PR DESCRIPTION
**Pull request by AnweshDas**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                click(item, focusedWindow){
+                    focusedWindow.toggleDevTools();
+                }
+            },
+            {
+              role: 'reload' 
+            }
+        ]
+    });
+}
```
Comment (shreyasbapat) [inline main.js:113]: A new line after this!

File: mainWindow.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   
+    <title>LoAdPy</title>
+</head>
+<body>
+    <h1>loadpy demo</h1>
+</body>
+</html>
```
Comment (shreyasbapat) [inline mainWindow.html:10]: Same here!

File: package.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "loadpy",
+  "version": "1.0.0",
+  "description": "loadpy Desktop app",
+  "main": "main.js",
+  "scripts": {
+    "start": "electron ."
+  },
+  "author": "Anwesh Das",
```
Comment (shreyasbapat) [inline package.json:9]: Add a AUTHORS file and add your name to it! 
This will be a community project!

**Review by shreyasbapat — CHANGES_REQUESTED** [review]
I am leaving some comments,

Retitle the PR with a more suitable and apt string!

**Comment by shreyasbapat** [comment]
And as been said in the CONTRIBUTING.rst file, you did't create a new branch!

File: main.js (hunk 1 above)
Comment (AnweshDas) [reply main.js:113]: 113 ke baad?

**Pull request closed without merging**